tests/robot-tests/tests/admin/bau/legacy_releases.robot

```diff
@@ -33,36 +33,87 @@ Create legacy release
+
+    user checks page contains element    //*[@data-testid="other-release-item"]//a[text()="legacy release description"]
+
+    user checks element attribute value should be
+    ...    //*[@data-testid="other-release-item"]//a[text()="legacy release description"]    href    http://test.com/
+
+Navigate to admin dashboard to update legacy release
+    user opens publication on the admin dashboard    ${PUBLICATION_NAME}
+    user opens accordion section    ${PUBLICATION_NAME}
+    user clicks element    testid:Edit publication link for ${PUBLICATION_NAME}
+    user waits until page contains title caption    ${PUBLICATION_NAME}
+    user waits until h1 is visible    Manage publication    %{WAIT_SMALL}
+
 Update legacy release
     user clicks element    xpath://tr[1]//*[text()="Edit release"]
     user waits until modal is visible    Edit legacy release
     user clicks button    OK
     user waits until modal is not visible    Edit legacy release
     user waits until h1 is visible    Edit legacy release
-    user enters text into element    id:legacyReleaseForm-description    Test collection 2
-    user enters text into element    id:legacyReleaseForm-url    http://test-2.com
-    user enters text into element    id:legacyReleaseForm-order    2
+    user enters text into element    id:legacyReleaseForm-description    updated description
+    user enters text into element    id:legacyReleaseForm-url    http://test2.com
     user clicks button    Save legacy release
 
 Validate updated legacy release
     user waits until h1 is visible    Manage publication
     user checks element count is x    css:tbody tr    1
-    # Changing order to 2 should not actually change it to 2
-    # as there isn't another release to swap the order with.
     user checks results table cell contains    1    1    1
-    user checks results table cell contains    1    2    Test collection 2
-    user checks results table cell contains    1    3    http://test-2.com
+    user checks results table cell contains    1    2    updated description
+    user checks results table cell contains    1    3    http://test2.com
+
+Validate public frontend shows changes made to legacy release after saving publication
+    user clicks button    Save publication
+    user waits until modal is visible    Confirm publication changes
+    user clicks button    Confirm
+    user waits until modal is not visible    Confirm publication changes
+    user navigates to public frontend    ${PUBLIC_RELEASE_LINK}
+
+    user clicks element    testid:Expand Details Section See other releases (1)
+
+    user checks page contains element    //*[@data-testid="other-release-item"]//a[text()="updated description"]
+
+    user checks element attribute value should be
+    ...    //*[@data-testid="other-release-item"]//a[text()="updated description"]    href    http://test2.com/
+
+Navigate to admin dashboard to update legacy release
```

Comment:
This test case name appears twice - I'd just add ` again` on the end of this to avoid confusion when investigating failures